training-app/spec/models/user_spec.rb

```diff
@@ -5,12 +5,9 @@
 describe User do
   describe '.initialize' do
     it 'creates an user with first and last names' do
-      user = User.create(
-        email: 'test@wolox.com.ar',
-        password: '123123123',
-        password_confirmation: '123123123',
-        first_name: 'Test',
-        last_name: 'TestLastName'
+      user = described_class.create(
```

Comment:
Please leave them in column

Reply:
While I agree its best for readability, Rubocop complains if I actually do that